protocols/grpc/agent.proto

```diff
@@ -59,6 +59,11 @@ message CreateContainerRequest {
 	repeated Device devices = 4;
 	repeated Storage storages = 5;
 	Spec OCI = 6;
+
+	// This field is used to indicate if the container needs to join
+	// sandbox shared pid ns. This is needed since we clear out all the
+	// namespace paths before they are sent to the agent.
```

Comment:
Please be specific that it overrides NEWPID settings in container's OCI spec, not just the NEWPID namespace path.